### PR TITLE
Bump Vaadin to 23.2.0 & Project to 2.0.0-beta14

### LINKED ISSUE #140: ApexCharts broken since Vaadin version 23.2.0
**Describe the bug**
After upgrading from Vaadin flow 23.1.7 to 23.2.0 the charts implemented with ApexCharts v2.0.0-beta13 are not working anymore.

The following error is shown in the JS console in Chrome:
```
Uncaught ReferenceError: require is not defined
    at ApexChartsWrapper.ready (generated-flow-imports.ad03bda7.js:7529:622)
    at E.ready (FlowClient.947c8d40.js:3:77663)
    at ApexChartsWrapper._enableProperties (generated-flow-imports.ad03bda7.js:193:2486)
    at ApexChartsWrapper.connectedCallback (generated-flow-imports.ad03bda7.js:235:1559)
    at ApexChartsWrapper.connectedCallback (generated-flow-imports.ad03bda7.js:245:5028)
````

We are using different chart types, at least sparkline and donut were not working anymore.
Downgrading to Vaadin 23.1.7 fixed the issue.

**Expected behavior**
ApexCharts should work after upgrading to Vaadin 23.2.0

**Desktop (please complete the following information):**
 - OS: Mac and Windows
 - Browser: chrome
 - Version: 2.0.0-beta12 

**Smartphone (please complete the following information):**

### PR DESCRIPTION
**Pull request by TFyre**
* Update to Vaadin 23.2.0
* Fix onecolor & ShadyCSS
(Copied from #134)
* Bump project version 2.0.0-beta14
* Remove unnecessary clean of `node_modules`

Fixes #140

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -204,31 +204,6 @@
...
-                        <fileset>
-                            <directory>./node_modules</directory>
-                            <includes>
-                                <include>**/**</include>
-                            </includes>
-                            <followSymlinks>false</followSymlinks>
-                        </fileset>
-                    </filesets>
-                </configuration>
-            </plugin>
```
Comment (mrgreywater) [inline pom.xml:231]: Any reason why this is removed? It's actually quite useful as vaadin regularly breaks itself between updates if no vaadin:clean-frontend is performed. I would actually go as far as adding the new tsconfig.json to the list of files to be removed when cleaning the project. Either way, this change doesn't seem to be related to the PR title itself.

Comment (TFyre) [reply pom.xml:231]: I removed it to speed up the build process. I dont see the value in removing `node_modules` everytime when you are building the project. 

When you are building the project without `package*.json` it raises a warning:
```
[INFO] --- vaadin-maven-plugin:23.2.0:build-frontend (default) @ apexcharts ---
[INFO] Reflections took 203 ms to scan 76 urls, producing 2573 keys and 14623 values
[INFO] Scanning classes to find frontend configurations and dependencies...
[INFO] Visited 590 classes. Took 556 ms.
[INFO] Visited 104 classes. Took 41 ms.
[INFO] Unable to determine version information. No vaadin-core-versions.json found
[INFO] Running `npm install` to resolve and optionally download frontend dependencies. This may take a moment, please stand by...
[INFO] using 'C:\Program Files\nodejs\npm.cmd --no-update-notifier --no-audit --scripts-prepend-node-path=true --ignore-scripts install' for frontend package installation
[WARNING] package-lock.json is missing from this project. This may cause the npm package installation to take several minutes. It is recommended to keep the package-lock.json file persistently in your project. Please stand by...
[INFO] Frontend dependencies resolved successfully.
[INFO] Copying frontend resources from jar files ...
[INFO] Visited 77 resources. Took 218 ms.
[INFO] Copying project local frontend resources.
[INFO] Copying frontend directory completed.
[INFO] Reflections took 99 ms to scan 76 urls, producing 2573 keys and 14623 values
[INFO] Running Vite ...
[INFO] Build frontend completed in 20487 ms.
```

What do you think?

Comment (mrgreywater) [reply pom.xml:231]: I have a similar maven-clean-plugin include in my own vaadin projects, and it only runs if you execute 'maven clean'. If you just compile the project using 'maven compile' or 'maven build' , the node_modules folder stays as it is. Cleaning up the node_modules folder every build isn't right, I agree with that. The point of this plugin definition is that if the build is somehow broken, running 'maven clean install' will also remove all potentially broken generated files and npm packages, before building again.
Either way, I think this change belongs in a different PR, or at least a separate commit.

Comment (TFyre) [reply pom.xml]: Moved to #143

**Comment by klauss42** [comment]
As this is now merged in master, when will there be a release?

**Comment by TFyre** [comment]
> As this is now merged in master, when will there be a release?

https://vaadin.com/directory/component/apexchartsjs
Its released and available on the vaadin directory (we need to update github)
